samwise.js

```diff
@@ -2,14 +2,14 @@
  * Autoplay plugin - Automatically advance slideshow after N seconds
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.